packages/types/src/wallet/keplr.ts

```diff
@@ -115,6 +115,10 @@ export interface Keplr {
     chainId: string,
     contractAddress: string
   ): Promise<string>;
+  getSecret20ViewingKeyOrPermit(
```

Comment:
I think it should be "Snip20" instead of "Secret20"

Reply:
Also for all other instances of "Secret20"

Reply:
Although in the spec it is indeed "Secret20", will check and return here

Reply:
Yeah, seems it has to be "Secret20" for now

Reply:
yeah I wasn't sure. I would've also thought it should be `snip20`. But as you said it say `secret20` elsewhere